eth/tracers/firehose.go

```diff
@@ -1962,9 +1907,11 @@ func (f *Firehose) panicInvalidState(msg string, callerSkip int) string {
 
 // printBlockToFirehose is a helper function to print a block to Firehose protocl format.
 func (f *Firehose) printBlockToFirehose(block *pbeth.Block, finalityStatus *FinalityStatus) {
-	var buf bytes.Buffer
+	channelSize := int(math.Ceil(20000 * 8 / 6))
```

Comment:
That is not really `channelSize`. Further more, the idea is to take the real `bytes` size and not an hardcoded value here. `proto.Size(block)` should yield how many bytes the proto marshalling would give back.

Reply:
Also, account for the `FIRE BLOCK ....` format so that the full string would fit in the buffer. Come see me if unclear here.